Base/etc/FileIconProvider.ini

```diff
@@ -16,3 +16,22 @@ python=*.py
 sound=*.wav
 spreadsheet=*.sheets
 text=*.txt
+
+[Overrides]
```

Comment:
This feels very hackish. How about we just rename the files to `*.json` instead? :D

Reply:
Indeed, it is quite hackish... but can we really do that without breaking stuff, same for the "text" files in `/proc`? `/proc/uptime.txt`? :thinking:

This is just a nice-to-have ofc, we can also leave it at the generic file icon.

Reply:
Sure, it'll break some hard-coded paths here & there, but we can fix those :)
Everything the light touches is our kingdom!

Reply:
All I see here is `br /proc/all.html`